helpers/helpers.go

```diff
@@ -646,3 +645,26 @@ func ReadBytes(valFile string) ([]byte, error) {
 			strings.Join(splitVal[:len(splitVal)-1], ", "))
 	}
 }
+
+// ParseConnectionStr parses a string representing the source of OCSP responses
+// which can either be a file or the path to a DB (Sqlite, MySQL or PostgreSQL).
+// It returns the type of the connection string (e.g. "File" or "MySQL" etc.) as
+// well as the path to the source.
+func ParseConnectionStr(conn string) (string, string, error) {
```

Comment:
This should be `ParseConnString` (or `NewSourceFromConnString` should be renamed `NewSourceFromConnectionString`).